charts/victoria-metrics-operator/templates/crd.yaml

```diff
@@ -1,18 +0,0 @@
-{{- /* do not update crds here, please update in /victoria-metrics-operator/crd.yaml */ -}}
```

Comment:
It's breaking change that will have strongly negative impact on chart users.

 Helm cannot perform CRD updates according to the documentation:
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations

 And it was the main point of keeping CRDs at templates.

Reply:
I had the same wrong impression( but resources with annotation `helm.sh/resource-policy: keep` won't be upgraded either during `helm upgrade`. So this pull request doesn't bring breaking change.
>The annotation helm.sh/resource-policy: keep instructs Helm to skip deleting this resource when a helm operation (such as helm uninstall, helm upgrade or helm rollback) would result in its deletion.

Reply:
We might need to add some instructions to explain this and tell users how to upgrade crds for operator and k8s-stack charts, wdyt @AndrewChubatiuk

Reply:
i've just checked helm code, this annotations prevents removal during upgrade, but seems like resource update itself should happen
https://github.com/helm/helm/blob/49819b4ef782e80b0c7f78c30bd76b51ebb56dc8/pkg/kube/client.go#L214

Reply:
Sounds like a bug at helm side. Because documentation wording mentions only `deleting` resource case. It should be upgrade on regular basis.

Reply:
> Helm cannot perform CRD updates according to the documentation
but crds are in templates in operator chart only, k8s-stack has them in `crds` folder of a subchart
I can update PR and leave it for user to decide if he wants crd as templates or as immutable object

Reply:
>Because documentation wording mentions only deleting resource case. It should be upgrade on regular basis.

Sorry, I did the wrong test, that's true and we shouldn't change it for operator chart.

Reply:
@f41gh7 @Haleygo 
reverted templating back, added boolean`crds.plain` instead, which is set to `false` by default to render CRDs from templates or when it's set to `true` it renders crds subchart `crds` folder